Formula/braid.rb

```diff
@@ -0,0 +1,68 @@
+class Braid < Formula
+  desc "Simple tool to help track vendor branches in a Git repository"
+  homepage "https://cristibalan.github.io/braid/"
+  url "https://rubygems.org/gems/braid-1.0.21.gem"
```

Comment:
Yes, me too. I found a problem with that because the gemspec relies on git commands to build up its manifest. If we use the release tarball we end up with an empty installation from the git commands returning nothing (no .git dir in the tarball). I could path the gemspec to work around this if you'd like. Alternatively pulling from the git tag works as well.